endoreg_db/data/report_reader_flag/rkh-histology-generic.yaml

```
@@ -0,0 +1,10 @@
+
+- model: endoreg_db.report_reader_flag
+  fields:
+    name: "rkh-histology-generic-cut-off-below-line"
+    value: "Mitkollegialen Grüßen"
+
+- model: endoreg_db.pdf_type
+  fields:
+    name: "rkh-histology-generic-patient-info-line"
+    value: "Pat.: "
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)